README.md

```diff
@@ -1 +1,70 @@
-# math-magician
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# React Project: Math Magician
```

Comment:
- [ ] You are yet to delete all unnecessary boilerplates files. Kindly delete the `public/robots.txt` and `src/setupTests.js` files. This way we rid our project of unnecessary files and keep it clean.

Reply:
@TedLivist The README file is part of Microverse Template. Should I remove that too?